lsp/src/main/kotlin/org/gradle/declarative/lsp/DeclarativeTextDocumentService.kt

```diff
@@ -143,6 +140,15 @@ class DeclarativeTextDocumentService : TextDocumentService {
 
     override fun didSave(params: DidSaveTextDocumentParams?) {
         LOGGER.trace("Saved document: {}", params)
+
+        params?.let {
+            val uri = URI(it.textDocument.uri)
+            // If the document is a settings file, we make an attempt updating the model
+            if (uri.path.endsWith("settings.gradle") || uri.path.endsWith("settings.gradle.kts")) {
```

Comment:
❓ Why is `settings.gradle.dcl` not among the filenames?

Reply:
Correct, _how could I even forget?!_ 😀

Reply:
Added